src/back/Tile.js

```diff
@@ -31,7 +31,7 @@ Tile.prototype.render = function (project, map, cb) {
     this.setupBounds();
     map.zoomToBox(this.projection.forward([this.minX, this.minY, this.maxX, this.maxY]));
     var im = new mapnik.Image(this.height, this.width);
-    map.render(im, {scale: project.mml.scale}, cb);
+    map.render(im, {scale: project.mml.scale, variables: {zoom: this.z}}, cb);
```

Comment:
Probably a good idea, but I haven't setup a vector source, so cannot test. Related: what about the scale factor for vector tiles?